tests/framework/validators/UniqueValidatorTest.php

```diff
@@ -331,6 +331,31 @@ public function testPrepareParams()
         $result = $this->invokeMethod(new UniqueValidator(), 'prepareConditions', [$targetAttribute, $model, $attribute]);
         $expected = ['val_attr_b' => 'test value b', 'val_attr_c' => 'test value a'];
         $this->assertEquals($expected, $result);
+
+        // Add table prefix for column name.
+        $model = Profile::findOne(1);
+        $attribute = 'id';
+        $targetAttribute = 'id';
+        $result = $this->invokeMethod(new UniqueValidator(), 'prepareConditions', [$targetAttribute, $model, $attribute]);
+        $expected = [Profile::tableName() . '.' . $attribute => $model->id];
+        $this->assertEquals($expected, $result);
+    }
+
+    public function testGetTargetClass()
+    {
+        // Filled property targetClass.
+        $validator = new UniqueValidator();
+        $validator->targetClass = 'target class';
+        $model = new FakedValidationModel();
+        $actualTargetClass = $this->invokeMethod($validator, 'getTargetClass', [$model]);
+        $this->assertEquals('target class', $actualTargetClass);
+
+        // Not filled property targetClass.
```

Comment:
you are preparing a new test scenario here. Please move this to a dedicated test method with a corresponding test name